### PR TITLE
Single Video Page

### PR DESCRIPTION
**Pull request by vijaytembugade**
- Added single video showing page
- Added video suggestions
- Basic Styling and `react-player`

### CODE REVIEW

File: src/Pages/VideoDetails/VideoDetails.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  }, [video, setSuggestions]);
+
+  console.log(suggestions);
+
+  return (
+    <>
+      <span
+        title="videos"
+        class="material-icons back-btn btn-float btn-secondary"
+        onClick={() => navigate(-1)}
```
Comment (atharvabhanage02) [inline src/Pages/VideoDetails/VideoDetails.js:41]: Liked the use of -1 in navigate

**Review by atharvabhanage02 — COMMENTED** [review]
- UI looks clean
- Code is well structured
- `Single Video Page`  functionality is working fine

**Review by atharvabhanage02 — APPROVED** [review]

**Review by nisargpawade — APPROVED** [review]
It looks good

**Review by narharikale — APPROVED** [review]
great work Vijay, liked how you created separate component

File: src/Componets/VideoDescription/VideoDescription.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./VideoDescription.css";
+
+const VideoDescription = ({ video }) => {
```
Comment (t007rushi) [inline src/Componets/VideoDescription/VideoDescription.js]: Destructure here for reading purpose

File: src/Pages/VideoDetails/VideoDetails.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    (async () => {
+      const { data } = await getSingleVideoService(id);
+      setVideo(data.video);
+    })();
+  }, [id]);
+
+  useEffect(() => {
+    (async () => {
+      const suggested = await getSuggestionsVideo(
+        video.author,
```
Comment (t007rushi) [inline src/Pages/VideoDetails/VideoDetails.js:26]: Same here

**Review by t007rushi — APPROVED** [review]
How are using both embedded and rectplayer at a time?

**Pull request merged**